mon-pix/tests/integration/components/staging-banner-test.js

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupRenderingTest } from 'ember-mocha';
+import { find, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const STAGING_BANNER_CONTAINER = '.staging-banner';
+
+describe('Integration | Component | staging-banner', function() {
+  setupRenderingTest();
+
+  describe('isStagingApp', function() {
+    it('false: it should not display the banner', async function() {
```

Comment:
La lecture des tests donne l'impression que l'on est dans un test unitaire.
Peut-être `it('should not display the banner when not in staging'`. et `it('should display the banner when in staging'`
Sans le `describe ('isStagingApp'` du coup.